pages/common/bioradtopgm.md

```diff
@@ -0,0 +1,12 @@
+# bioradtopgm
+
+> Convert a Biorad confocal file into a PGM file.
+> More information: <https://netpbm.sourceforge.net/doc/bioradtopgm.html>.
+
+- Read a Biorad confocal file and store the n'th image contained in it to as a PGM file:
+
+`bioradtopgm -image {{n}} {{path/to/input_file}} > {{path/to/output_file.pgm}}`
```

Comment:
```suggestion
`bioradtopgm -{{n}} {{path/to/file.pic}} > {{path/to/file.pgm}}`

- Read a Biorad confocal file and print the number of images it contains:

`bioradtopgm {{path/to/file.pic}}`
```

After having read the source code, I have finally understood what the options are supposed to be like. I have attacked a suggestion which adds examples.

Reply:
Thank you so much for the insights!
I've made the suggest amendments in the latest commit,
Please do let me know If anything else is to be added